jobs/tests/data/inventory-simple.csv

```diff
@@ -7,3 +7,4 @@
 "net-mozaws-prod-delivery-firefox","pub/firefox/candidates/55.0b9-candidates/build2/win64/zh-TW/firefox-55.0b9.zip","53251778","2017-07-13T23:37:21.000Z","0315517f673697841a5e28ff44da2eb9"
 "net-mozaws-prod-delivery-firefox","pub/firefox/candidates/55.0b9-candidates/build2/win64/zh-TW/Firefox+Setup+55.0b9.exe","37219504","2017-07-13T23:37:23.000Z","ff391f7201575ab2ba518fce20d45596"
 "net-mozaws-prod-delivery-archive","pub/devedition/candidates/55.0b1-candidates/build5/win64/pt-BR/firefox-55.0b1.zip","53718907","2017-06-14T00:41:56.000Z","8eb5c067a880ca9c63a1bf0a6c9a631d"
+"net-mozaws-prod-delivery-firefox","pub/firefox/releases/51.0b11/mac-EME-free/mr/Firefox+51.0b11.dmg","85984611","2017-01-04T00:05:18.000Z","9b4e5eccde90dcfd45730f810f5efc53"
```

Comment:
Why was this skipped?

Reply:
Because `is_build_url()` returns `None` for `https://archive.mozilla.org/pub/firefox/releases/51.0b11/mac-EME-free/mr/Firefox+51.0b11.dmg`

Reply:
Didn't we decided to ignore EME-free builds?

Reply:
Problems comes from `+` signs in inventories. HUUUUUUGE fix coming in this PR :D